csvqb/csvqb/tests/behaviour/qbwriter.feature

```diff
@@ -1,15 +1,15 @@
 Feature: Test outputting CSV-Ws with Qb flavouring.
 
   Scenario: A QbCube should generate appropriate DCAT Metadata
-    Given a single-measure QbCube named "Some Qube"
+    Given a single-measure QbCube with identifier "qb-id-10002" named "Some Qube"
```

Comment:
If an identifier is not given, would we have a QbCube with a title?

Reply:
Yes, `title` is a required property. And if the user doesn't provide it, then they're ignoring all of our advice.